config/interpolate_funcs.go

```diff
@@ -23,6 +23,7 @@ func init() {
 		"element": interpolationFuncElement(),
 		"replace": interpolationFuncReplace(),
 		"split":   interpolationFuncSplit(),
+		"toint":   interpolationFuncToint(),
```

Comment:
I think it might be clearer if this was `to_int` for UX. "toint" sounds like a single word to be and could be confusing. What do you think?